ompi/op/op.h

```diff
@@ -512,11 +523,9 @@ static inline void ompi_op_reduce(ompi_op_t * op, const void *source,
      * in iterations of counts <= INT_MAX since it has an `int *len`
      * parameter.
      *
-     * Note: When we add BigCount support then we can distinguish between
-     * a reduction operation with `int *len` and `MPI_Count *len`. At which
-     * point we can avoid this loop.
      */
-    if( OPAL_UNLIKELY(full_count > INT_MAX) ) {
```

Comment:
The cast to `int` I mentioned on the previous ticket is still problematic (https://github.com/open-mpi/ompi/pull/13030/files#diff-dfd595860dc4d0c9c73114bd8563c0786d063327f6e30955a3e361c3f6098e3dR519). I can file a PR to fix if you prefer handling that separately.

Reply:
please do.

Reply:
Will do once this one is in.

Reply:
okay moved that assert ahead of the cast.